Library/Formula/sfml.rb

```diff
@@ -27,9 +17,41 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Remove the bundled libs wording that causes linking errors.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  patch :DATA
 
   def install
-    system "cmake", ".", "-DCMAKE_INSTALL_FRAMEWORK_PREFIX=#{frameworks}/", *std_cmake_args
+    args = std_cmake_args
+    args << "."
+
+    args << "-DSFML_BUILD_DOC=TRUE" if build.with? "doxygen"
+
+    # If we leave this dir present, sfml will always prefer to use its bundled deps
+    # instead of ours, and that breaks absolutely everything.
+    rm_rf buildpath/"extlibs"
```

Comment:
We could always add a note to the removal line to encourage whomever updates this formula next to download the untouched upstream zip and review it for new deps, if that's going to be an immediate change with the next release.